File: DailyFlash Solutions/DailyFlash Solutions/03-DailyFlash_Solutions/16_Jan_Solutions_One/CPP/pattern.cpp
/*
Program 4: Write a Program to Print following Pattern
1  2  3  4
5  6  7  8
9  10 11 12
13 14 15 16
*/


#include<iostream>
#define UPPERLIMIT 4
#define LOWERLIMIT 1

int main(){
	
	int display = 0;
	for(int outerItr = LOWERLIMIT ; outerItr <= UPPERLIMIT ; outerItr++){
		
		for(int innerItr = LOWERLIMIT ; innerItr <= UPPERLIMIT ; innerItr++){
			
			std::cout<<++display<<"\t";

		}	
		std::cout<<"\n";
	}	


}	

File: DailyFlash Solutions/DailyFlash Solutions/04-DailyFlash_Solutions/17_Jan_Solutions_One/Python/pattern.py
"""
Program 4: Write a Program to Print following Pattern
2 4 6 8
2 4 6 8
2 4 6 8
2 4 6 8

"""

for outer in range(1,5):
    for inner in range(1,5):
        print(inner*2,end=' ')
    print(" ")    
